features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/operationUtils.js

```diff
@@ -85,4 +85,42 @@ function getTaggedOperations(api, openAPI) {
     }
 }
 
-export { getTaggedOperations, getAPIProductTaggedOperations };
+/**
+ *Extract the path parameters from URI template. User has to give the Open API spec version as well
+ * https://github.com/OAI/OpenAPI-Specification/tree/master/versions
+ * @param {String} target URI template
+ * @param {String} openAPIVersion Should be a valid Open API specification version (i:e "2.0", "3.0.0")
+ * @returns {Array} List of parameter objects according to the given spec version
+ */
+function extractPathParameters(target, openAPIVersion) {
+    const regEx = /[^{}]+(?=})/g;
+    const params = target.match(regEx) || [];
+    let parameters = [];
+    if (['3.0.0', '3.0.1', '3.0.2'].includes(openAPIVersion)) {
```

Comment:
Shall we modify this condition as,
if ('2.0'.includes(openAPIVersion) {
} else {
}?

Otherwise, we have to keep incrementing the supported openApi v3.x.x list.

Reply:
Yes , That is intended, Because if 3.0.3 spec version introduced something we haven't considered or handle, it will break the UI